sdk/typescript/src/transactions/index.ts

```diff
@@ -47,3 +47,5 @@ export type {
 } from './json-rpc-resolver.js';
 
 export { Arguments } from './Arguments.js';
+
+export { nameResolutionPlugin, type NameResolutionPlugin } from './plugins/NameResolutionPlugin.js';
```

Comment:
I think this name might be a little too generic

Reply:
OK I'd agree. I changed to `NamedPackagesPlugin` now.